### DISCUSSION TITLE
Fermata AAwireless issue

### DISCUSSION THREAD

**Discussion by Blabze** [AndreyPavlenko/Fermata#504] [General]
Have S22 phone with A14 & UI6.1, AA wireless with developper mode enabled in his app companion.
Fermata installed with king installer.
But android auto in car throught AAwireless show only fermata control or fermata media services in Last version 1.9.9.
Any help please?

**Comment by AndreyPavlenko** [comment]
Seems very similar to https://github.com/AndreyPavlenko/Fermata/issues/503

**Comment by Blabze** [comment]
Thank you Andrey, yes same issue, i searched before but dont find this #503

**Comment by Blabze** [comment]
After some tests, I managed to get the 3 app fermata 1.9.9
-Enable fermata in phone acessibilty setting.
-Enable Carplay BETA, then enable AA to CP in AAwireless app settings, dev mode enabled.
AA in car will restart in Carplay mode and fermata apps are shown and working well (5min tests).
The screen resolution in Carplay mode is lower and the characters are very small, is there a setting to improve that?
I will leave AAwirelees in AA mode and when I need fermata apps, I will launch Carplay mode.
I hope the AAwirelees team will solve this issue in AA mode.
